starter-code/chronometer.js

```diff
@@ -0,0 +1,54 @@
+function Chronometer(minutes, seconds) {
+  this.minutes = minutes;
+  this.seconds = seconds;
+}
+
+Chronometer.prototype.startClick = function() {
+
+  document.getElementById("btnLeft").innerHTML = "STOP";
+  document.getElementById("btnLeft").className = "btn stop"
+}
+
+Chronometer.prototype.stopClick = function() {
+
+  document.getElementById("btnLeft").innerHTML = "START";
+  document.getElementById("btnLeft").className = "btn start"
+}
+
+Chronometer.prototype.resetClick = function() {
+  document.getElementById("btnRight").innerHTML = "SPLIT";
+  document.getElementById("btnRight").className = "btn split"
+}
+
+Chronometer.prototype.splitClick = function() {
+  document.getElementById("btnRight").innerHTML = "RESET";
+  document.getElementById("btnRight").className = "btn reset"
+}
+
+
+function seconds() {
+  var i = 0;
+  var minutes = 0;
+  var counterSeconds = setInterval(function() {
+    console.log(i);
+    i++;
+    if (i == 10) {
+      i=0;
+      minutes +=1;
+      console.log("Minutes"+minutes)
+    }
+  }, 1000);
+}
+
```

Comment:
Recordad, el número que marca el inicio de vuestro contador no es una variable i, sino que es la propiedad this.seconds. Por ejemplo, se puede hacer algo así:


Chronometer.prototype.startcount = function () {
  var that = this;
  this.intervalId = setInterval(function () {
    that.seconds += 1;
    that.printTime();
  }, 1000);